src/renderer/api-decorator.js

```diff
@@ -240,7 +240,7 @@ limitations under the License.
 
         // main window
         if (uuid === name) {
-            eventMap.push([`application/initialized/${uuid}`, undefined]);
+            eventMap.push([`application/initialized/${uuid}`, winIdentity]);
```

Comment:
Can you confirm that this doesn't change the payload of application's "initialized" event? It's supposed to only contain the uuid.